wordpress-mobile/WordPress-Android
# Implement composable UI for Dynamic Dashboard Cards

Fixes #19735

This PR includes composable UI for the dynamic dashboard cards feature only. Unfortunately, there is no way to load images in the preview mode, so I created placeholders for icons and feature images. To test with  We should consider if we are good to have such placeholders or modify them.

<img width="650" alt="Screenshot 2023-12-12 at 3 06 34 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/ed1cb35d-06ca-44f2-a1c6-ebbf6573e99b">

-----

## To Test:

Go to `org/wordpress/android/ui/mysite/cards/dynamiccard/DynamicCardComposable.kt` file in AndroidStudio and make sure the cards preview look accordingly to the designs: YSMw2nbFAgPpjZLLbYvTfT-fi-1281_3822

-----

## Regression Notes

1. Potential unintended areas of impact

    - N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - N/A

3. What automated tests I added (or what prevented me from doing so)

    - N/A

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
